dev/docker_data/default_pipeline.yml

```diff
@@ -398,6 +398,53 @@ regWithSkull :  [1]
 runSegmentationPreprocessing :  [1]
 
 
+# ANTs Prior-based Segmentation workflow that has shown optimal results for non-human primate data.
+# Generate white matter, gray matter, CSF masks based on antsJointLabelFusion
+# If you'd like to set ANTs Prior-based Segmentation as your segmentation option,
+# please set Template based segmentation as 'None' and runSegmentationPreprocessing as [0].
+ANTs_prior_based_segmentation : [0]
+
+
+# The atlas image assumed to be used in ANTs Prior-based Segmentation.
+ANTs_prior_seg_template_brain_list : 
+  - /monkey_seg/templates/JointLabelCouncil/MacaqueYerkes19_T1w_0.5mm/T1w_brain.nii.gz
```

Comment:
Are these something we're going to bundle in the container, in /monkey_seg?
And how is it going to be encoded in the Dockerfile?

Reply:
yep. probably we could upload these templates to s3 ?  Same as ants oasis template?   [https://github.com/FCP-INDI/C-PAC/blob/feature/antsprior_seg/Dockerfile#L137-L142](url)

Reply:
Sure, sounds good. Would reach out to @hahaai for that. It would be good to see if we can have them in the same overarching directory like "cpac resources", I believe it is.